packages/service-core/src/routes/endpoints/sync-stream.ts

```diff
@@ -21,6 +21,9 @@ export const syncStreamed = routeDefinition({
   validator: schema.createTsCodecValidator(util.StreamingSyncRequest, { allowAdditional: true }),
   handler: async (payload) => {
     const system = payload.context.system;
+    const headers = payload.request.headers;
+    const userAgent = headers['x-user-agent'] ?? headers['user-agent'];
```

Comment:
Are there any limitations for adding this to the websocket method? I think we can access headers in the initial request or add this data to the request itself.

Reply:
I added this as a parameter to the setup metadata now.

Reply:
👍